css_tasks/task5/index.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Task5</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Bitter|Cinzel:700,900|Ubuntu:300,400,700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="main.css" />
+  </head>
+  <body>
+    <!-- INPUTS TO CHANGE DEFAULT STYLES -->
+
+    <div class="container">
+    <label for="theme-toggler" class="container__toggler toggler--theme">Change Theme</label>
+    <input type="checkbox" id="theme-toggler" class="theme-toggler">
+    <label for="font-toggler" class="container__toggler toggler--font">Change Font</label>
```

Comment:
by BEM you can't have modifier class if you don't have base class, so you need to write is just as class with one `-` or add base class which is toggler

Reply:
ok